clouddriver-oracle-bmcs/src/main/groovy/com/netflix/spinnaker/clouddriver/oraclebmcs/cache/Keys.groovy

```diff
@@ -103,6 +112,13 @@ class Keys {
     "$OracleBMCSCloudProvider.ID:${Namespace.IMAGES}:${account}:${region}:${imageId}"
   }
 
+  static String getInstanceKey(String account,
+                               String region,
+                               String name,
+                               String id) {
+    "$OracleBMCSCloudProvider.ID:${Namespace.INSTANCES}:${account}:${region}:${name}:${id}"
```

Comment:
Woops, no it's not intentional - i've updated the switch statement on line 60/61